### DISCUSSION TITLE
Integrate the Flow CLI into Flowser

### DISCUSSION THREAD

**Discussion by devbugging** [onflowser/flowser#84] [Ideas]
It would be cool to have support for the Flow CLI in the flowser

**Comment by bartolomej** [comment]
UPDATE: We are now planning to develop a desktop app and integrate Flowser into FLOW CLI. (https://github.com/onflow/developer-grants/issues/27)

Mini feature idea: Check for flow-cli installation when launching Flowser desktop app. Users that will encounter Flowser before Flow CLI could appreciate that + it would be a usefull safety check.

**Comment by bartolomej** [comment]
Flowser is now available within Flow CLI, as of https://github.com/onflow/flow-cli/releases/tag/v0.42.0.
